src/views/ModelPlan/TaskList/ITTools/PageFive/index.test.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { MockedProvider } from '@apollo/client/testing';
+import { render, waitFor } from '@testing-library/react';
+
+import GetITToolsPageFive from 'queries/ITTools/GetITToolsPageFive';
+import {
+  GetITToolPageFive_modelPlan_itTools as GetITToolsPageFiveType,
+  GetITToolPageFive_modelPlan_opsEvalAndLearning as OpsEvalAndLearningType
+} from 'queries/ITTools/types/GetITToolPageFive';
+import { OelProcessAppealsType } from 'types/graphql-global-types';
+
+import ITToolsPageFive from '.';
+
+const itToolsPageFiveMockData: GetITToolsPageFiveType = {
+  __typename: 'PlanITTools',
+  id: '',
+  oelProcessAppeals: [OelProcessAppealsType.MEDICARE_APPEAL_SYSTEM],
+  oelProcessAppealsOther: '',
+  oelProcessAppealsNote: '',
+  oelEvaluationContractor: [],
+  oelEvaluationContractorOther: '',
+  oelEvaluationContractorNote: '',
+  oelCollectData: [],
+  oelCollectDataOther: '',
+  oelCollectDataNote: ''
+};
+
+const opsEvalAndLearningMockData: OpsEvalAndLearningType = {
+  __typename: 'PlanOpsEvalAndLearning',
+  id: '',
+  appealPerformance: true,
+  appealFeedback: null,
+  appealPayments: null,
+  appealOther: null,
+  evaluationApproaches: [],
+  dataNeededForMonitoring: []
+};
+
+const itToolsPageFiveMock = [
+  {
+    request: {
+      query: GetITToolsPageFive,
+      variables: { id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905' }
+    },
+    result: {
+      data: {
+        modelPlan: {
+          id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905',
+          modelName: 'My excellent plan that I just initiated',
+          itTools: itToolsPageFiveMockData,
+          opsEvalAndLearning: opsEvalAndLearningMockData
+        }
+      }
+    }
+  }
+];
+
+describe('IT Tools Page five', () => {
+  it('renders without errors', async () => {
+    const { getByTestId } = render(
+      <MemoryRouter
+        initialEntries={[
+          '/models/ce3405a0-3399-4e3a-88d7-3cfc613d2905/task-list/it-tools/page-five'
+        ]}
+      >
+        <MockedProvider mocks={itToolsPageFiveMock} addTypename={false}>
+          <Route path="/models/:modelID/task-list/it-tools/page-five">
+            <ITToolsPageFive />
+          </Route>
+        </MockedProvider>
+      </MemoryRouter>
+    );
+
+    await waitFor(() => {
+      expect(getByTestId('it-tools-page-five-form')).toBeInTheDocument();
+    });
+
+    await waitFor(() => {
+      expect(getByTestId('tools-question-appealPerformance')).toHaveTextContent(
+        'Participants will be able to appeal the following'
```

Comment:
I noticed this question on page 5 doesn't show which of the options were selected. Is that as intended? I couldn't tell, I could see that being intentional.

In the case below, I've only selected yes for payments

![image](https://user-images.githubusercontent.com/99294987/180562810-fd97de9e-7e36-4718-9581-5e828baf1106.png)

Reply:
Ah yeah, I need to sync with Natasha on this. 'Payments' would be the only answer shown if the others weren't answered.  I think you answered 'No' to the other questions, so it would still show those as answered on IT Tools.  Maybe have a. 'Yes/No' next to each answer would make more sense?  I'll update with comment after syncing with Natasha

Reply:
Sync'd with Natasha, I'll need to change this to only render questions that have 'Yes' checked, even if 'No' was answered.